tst/server/User.test.ts

```diff
@@ -8,3 +13,25 @@ test("invalid username", () => {
     expect.assertions(1);
     return expect(getAlteredUsername("")).rejects.toThrowError("Invalid username");
 });
+
+// Begin addUser test cases
+describe("addUser() tests", () => {
+    test("valid volunteer", async () => {
+        const mockVol: Volunteer = {
+            email: "test",
+            name: "test",
+            phone: "123-123-test",
```

Comment:
We'll eventually want to run some regex on the phone number to see if it's valid syntax before adding it to the db. Can you change the "test" to numbers?